.jenkins/pytorch/win-build.sh

```diff
@@ -91,6 +91,7 @@ if "%REBUILD%"=="" ( call conda install -y -q numpy cffi pyyaml boto3 )
 :: Install ninja
 if "%REBUILD%"=="" ( pip install ninja )
 
+call "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Auxiliary\\Build\\vcvarsall.bat" x64
 call "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Auxiliary\\Build\\vcvarsall.bat" x86_amd64
```

Comment:
What you really wanted to do here was just change `x86_amd64` to `amd64`, which is the 64-bit native toolchain.

Reply:
Yes, I tried that first. But it seems that the x64 compiler has some bug in it and cannot complete the build process. See the [log](https://ci.pytorch.org/jenkins/job/pytorch-builds/job/pytorch-win-ws2016-cuda9-cudnn7-py3-trigger/23914/) for details. So I switched back to x86_amd64 and only added x64 toolchain into PATH by activating it first.
You may say that this line is useless in your own environment. But this one is necessary here. Because there is another `link.exe` in `PATH` (used by git bash), it will cause the auto fallback mechanism to fail. By activating the x64 env, the x64 linker will be listed before the linker provided by git bash in `PATH`, thus the problem is solved.